diff --git i/snf-image-host/common.sh.in w/snf-image-host/common.sh.in
index 8c9f72a..1fd7adf 100644
--- i/snf-image-host/common.sh.in
+++ w/snf-image-host/common.sh.in
@@ -25,6 +25,7 @@ SFDISK="sfdisk"
 INSTALL_MBR="install-mbr"
 TIMEOUT="timeout"
 CURL="curl"
+WGET="wget"
 TAR="tar"
 DATE="date -u" # Time in UTC
 KVM="kvm"
diff --git i/snf-image-host/snf-image-update-helper.in w/snf-image-host/snf-image-update-helper.in
index 5c52d00..8870e17 100644
--- i/snf-image-host/snf-image-update-helper.in
+++ w/snf-image-host/snf-image-update-helper.in
@@ -111,7 +111,7 @@ echo >&2
 echo "Downloading helper image from $HELPER_URL ..." >&2
 add_cleanup rm -f "$IMAGE_DIR/$IMAGE"
 set +e
-$CURL -O -L -f "$HELPER_URL"
+$WGET "$HELPER_URL"
 RET=$?
 set -e
 
@@ -127,7 +127,7 @@ if [ "x$NO_CHECKSUM" != "xyes" ]; then
     echo "Checking the helper image checksum ..." >&2
     add_cleanup rm -f "$IMAGE_DIR/$IMAGE.md5sum"
     set +e
-    $CURL -O -L -f "$HELPER_URL.md5sum"
+    $WGET "$HELPER_URL.md5sum"
     RET=$?
     set -e
 
